pkg/actuators/client/client.go

```diff
@@ -536,17 +536,19 @@ func (c *ibmCloudClient) GetSubnetIDbyName(subnetName string, resourceGroupID st
 	subnetOption.SetResourceGroupID(resourceGroupID)
 
 	// Get a list of all subnets
-	subnetList, _, err := c.vpcService.ListSubnets(subnetOption)
+	subnetPager, err := c.vpcService.NewSubnetsPager(subnetOption)
+	if err != nil {
+		return "", err
```

Comment:
Should the error be wrapped with additional context?
```suggestion
		return "", fmt.Errorf("could not get subnet pager: %w", err)
```

Reply:
Yep, I can wrap the error....

Reply:
Updated